ios/RNCPagerView.h

```diff
@@ -5,23 +5,19 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@interface ReactNativePageView: UIView
+@interface RNCPagerView: UIView
 
-- (instancetype)initWithEventDispatcher:(RCTEventDispatcher *)eventDispatcher;
+- (instancetype)initWithEventDispatcher:(id<RCTEventDispatcherProtocol> )eventDispatcher;
 
 @property(nonatomic) NSInteger initialPage;
-@property(nonatomic) NSInteger lastReportedIndex;
-@property(nonatomic) NSInteger destinationIndex;
+@property(nonatomic) NSString* orientation;
 @property(nonatomic) NSInteger currentIndex;
 @property(nonatomic) NSInteger pageMargin;
```

Comment:
I think this can be removed, because it's not used anymore inside `RNCPagerView.m`